### DISCUSSION TITLE
Re-upload ready-made vst files

### DISCUSSION THREAD

**Discussion by KastomKnuckles** [jpcima/ysfx#74] [General]
I don't have the ability to compile vst from source, and I would like to download a ready-made solution

**Comment by JoepVanlier** [comment]
Hey, I'm not the original author of this repo, but I've forked the project and compiled some of the binaries and attached the artefacts to a release. If you want the last state as the plugins were when development paused/stopped, then use [v0.02 here](https://github.com/JoepVanlier/ysfx/releases/tag/v0.0.2).

Note that quite a few of the more recent plugins don't work on that build though.

I have since been updating it with some more recent JSFX features, so if you want to load later newer plugins, you can grab artefacts from the latest release here: https://github.com/JoepVanlier/ysfx/tags but be aware that all of that should be considered pre-release. Bug reports are welcome, but be mindful that I don't have that much time to work on this.
